.node-version

```diff
@@ -1 +1 @@
-16.10.0
+18
```

Comment:
sneak in this change, so our node version manager switches node version automatically

Reply:
ah thanks for catching that, I missed it.